networkdb/cluster.go

```diff
@@ -107,7 +107,6 @@ func (nDB *NetworkDB) clusterInit() error {
 		config.BindPort = nDB.config.BindPort
 	}
 
-	config.ProtocolVersion = memberlist.ProtocolVersionMax
```

Comment:
Looking at the code setting explicitly the version does not really imply much unless you want to force a non compatible version bump.
Each node calculates the protocol min and protocol max between all the nodes using the protocolMin and ProtocolMax values, then checks that the current version of the node is within the range.
17.05 has a range between 1 and 3, with the memberlist bump we are between 1 and 5, if we keep the memberlist updated It will be undesirable that they will bump the default value to 6 suddenly.
What do you think? Do you still think it makes sense to have it explicit?